_pydev_bundle/pydev_monkey.py

```diff
@@ -1191,6 +1194,16 @@ def pydev_start_new_thread(self, function, args=(), kwargs={}):
             """
             return _original_start_new_thread(_UseNewThreadStartup(function, args, kwargs), ())
 
+    class ClassWithPydevStartJoinableThread:
+        def pydev_start_joinable_thread(self, function, *args, **kwargs):
+            """
+            We need to replace the original thread_module._start_joinable_thread with this function so that threads started
+            through it and not through the threading module are properly traced.
+            """
+            handle = kwargs.pop("handle", None)
```

Comment:
This doesn't seem right... what if the user passes things in `args` and not `kwargs`?

Why not just store the full args and kwargs and pass it?

Still, I'm not sure this patching is even needed on Python 3.13 anymore as the sys.monitoring API is global and no longer per thread as was with sys.settrace.

This needs a test that starts a thread with `_start_joinable_thread` and sees if breakpoints are hit in that thread (without the patching in place -- if it doesn't work then the patching is needed, but I'm not sure the current implementation works if args are not passed in kwargs).

Reply:
I guess we can leave as is for now and wait if someone complains...